MSAL/test/automation/ios/actions/MSALAutomationAcquireTokenSilentAction.m

```diff
@@ -81,11 +82,9 @@ - (void)performActionWithParameters:(MSIDAutomationTestRequest *)testRequest
         {
             NSError *error = MSIDCreateError(MSALErrorDomain, MSALErrorInteractionRequired, @"no account", nil, nil, nil, nil, nil);
 
-            NSString *errorName = MSALStringForErrorCode(MSALErrorInteractionRequired);
-
             result = [[MSIDAutomationErrorResult alloc] initWithAction:self.actionIdentifier
                                                                  error:error
-                                                             errorName:errorName
+                                                             errorName:@""
```

Comment:
How will automation test check the error?

Reply:
done